modules/dmrpp_module/build_dmrpp_util.cc

```diff
@@ -1080,12 +1119,29 @@ void get_chunks_for_all_variables(hid_t file, D4Group *group) {
             // that variable was synthesized (likely for CF compliance)
             H5Eset_auto2(H5E_DEFAULT, nullptr, nullptr);
             string FQN = (*btp)->FQN();
+            string name = (*btp)->name();
+            if (nc4_non_coord_candidate.empty() == false) {
```

Comment:
You don't need to test if the set is empty(). If it is, find() will return end().

Reply:
Remove the empty